tests/nnfw_api/src/GenModelTest.h

```diff
@@ -222,7 +222,7 @@ class GenModelTestContext
     for (auto backend : backends)
     {
 #ifdef TEST_ACL_BACKEND
-      if (backend == "acl_cl" || backend == "acl_neon")
+      if (backend == "acl_cl" || backend == "acl_neon" || backend == "gpu_cl")
```

Comment:
This ifdef block seems wired for gpu_cl test since TEST_ACL_BACKEND is a switch for Arm ComputeLibrary.  We need to make a new block for gpu_cl backend.

Reply:
i will add.